grpc/src/test/java/com/linecorp/armeria/internal/common/grpc/TestServiceImpl.java

```diff
@@ -365,7 +366,9 @@ private synchronized void dispatchChunk() {
                 }
             } catch (Throwable e) {
                 failure = e;
-                if (GrpcStatus.fromThrowable(e).getCode() == Status.CANCELLED.getCode()) {
+                if (GrpcExceptionHandlerFunction.ofDefault()
+                                                .apply(null, e, null)
```

Comment:
Global comment: Could you pass non-null context and metadata? Otherwise all your tests will get broken once we start to access them in the default handler function.

You can create a mock context from `{Client,Service}RequestContext.of()` and an empty `Metadata` simply by calling its default construtor.

Reply:
Applied feedback. Thanks